src/common/runtime/src/global.rs

```diff
@@ -19,13 +19,36 @@ use std::sync::{Mutex, Once};
 use common_telemetry::info;
 use once_cell::sync::Lazy;
 use paste::paste;
+use serde::{Deserialize, Serialize};
 
 use crate::{Builder, JoinHandle, Runtime};
 
 const READ_WORKERS: usize = 8;
 const WRITE_WORKERS: usize = 8;
 const BG_WORKERS: usize = 8;
```

Comment:
Do we still need them?

Reply:
Right now yes, although I think the global runtime init can be refactor to a more clear way. But that doesn't have to be in this PR.